sass/_forms.scss

```diff
@@ -337,7 +337,7 @@ $form-padding: 0.5rem;
       padding: 0;
     }
   }
-}
+} // /omnisearch
```

Comment:
that notation is my personal style choice 
`// /closes-the-block`
similar to an HTML closing block `</tag>` 
sometimes the nested braces get very hard to see / read and know at a glance where the block ends...